docs/source/resources/scyllaclusters/nodeoperations/volume-expansion.md

```diff
@@ -0,0 +1,69 @@
+# Resizing storage in ScyllaCluster
+
+
+::::{caution}
+The StorageClass used for your PersistentVolumeClaims must support volume expansion. To verify if your StorageClass supports resizing, run:
+
+```bash
+kubectl get storageclass
+```
+
+Check the `allowVolumeExpansion` column. It should be set to `true` for StorageClass used by ScyllaCluster.
+::::
+
+
+Let's use the following ScyllaCluster as an example.
+```bash
+$ kubectl get scyllacluster
+NAME     READY   MEMBERS   RACKS   AVAILABLE   PROGRESSING   DEGRADED   AGE
+scylla   3       3         1       True        False         False      62m
+```
+
+## Increasing the storage size
+
+1. Since we are going to delete the ScyllaCluster object, make sure to have a copy of the existing one. You will need it later.
+
+2. Orphan delete ScyllaCluster
```

Comment:
I think we could be more explicit and tell why we orphan-delete instead of cascading  (I believe it's because we'd like to keep manual control over what objects are deleted, as we want to preserve some of the ScyllaCluster's children). I suspect it may not be obvious to the reader.

Reply:
done